### ISSUE TITLE
a11y black and white terminal css template

### ISSUE THREAD

**Issue by ityonemo** [E-xyza/ex_term#9]

**Comment by ityonemo** [comment]
deppends on https://github.com/E-xyza/ex_term/issues/8

**Comment by ityonemo** [comment]
completed in [e05cf48](https://github.com/E-xyza/ex_term/commit/e05cf48c370031548ac0a370e6a70822a665b3a9)
